app/src/main/java/com/infomaniak/drive/ui/SaveExternalFilesActivity.kt

```diff
@@ -382,7 +392,7 @@ class SaveExternalFilesActivity : BaseActivity() {
     }
 
     private fun isValidFields(): Boolean {
-        return (isMultiple || !binding.fileNameEdit.showOrHideEmptyError()) &&
+        return (isMultiple || !binding.fileNameEdit.showOrHideEmptyError() || navigationArgs.isPublicShare) &&
```

Comment:
Why not just keep the same condition as before, since you're supposed to have a name every time, and so the name field should be completed?

Reply:
In fact, in the case of public share, this activity is only needed to chose a destination, not to really upload file.
Then it's directly the back that will import all the files of the share to the destination.
So we don't chose a file whose name could be displayed, and we cannot fetch the name from realm.